featureA/one-more-file.txt

```diff
@@ -0,0 +1,4 @@
+This file was also added in the feature1 branch.
```

Comment:
Perfect line added.
`print("What do you think?")`

Reply:
Thanks pointer.

Reply:
Thanks, pointer

Reply:
Bit a conflict with code and github app?!?